src/scrabbleGame.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+const tiles = require('./tiles')
+const readlineSync = require('readline-sync')
+const { yourTiles, scrabBag } = require('./scrabBag')
+
+const yourTurn = ['BCEED_F', 'AXYREAS', 'E_KIEUM']
+
+console.log("Hi let's play scrabble! \n Here are the tiles in your scrabblebag!)\n", tiles)
+console.log("Here are your first seven tiles: BCEED_F")
+
+const turn1 = readlineSync.question('ok play a word, hit enter when you are done: ')
+yourTiles(yourTurn[0])
+scrabBag(tiles, yourTiles(yourTurn[0]))
```

Comment:
Check spacing on your arguments, it's inconsistent compared to the scrabBag file.